src/js/main.js

```diff
@@ -56,8 +56,8 @@ Site.SplashVideo = {
     // TODO: Detect can autoplay
     _this.canAutoplay = true;
 
-    // TODO: Get threshold based on video position/margin/something
-    _this.threshold = 100;
+    // Get threshold based on video position/margin/something
+    _this.threshold = _this.$video1.offset().top;
```

Comment:
This feels to be too late. The video should start playing when the video has been 50% or 66% scrolled into view. Otherwise it is too easy to scroll thru the video without watching. 

(This is the moment that the triggers get weird tho because on the way back up the trigger moment will be really late. That is what Waypoints or similar is supposed to be able to solve. But the first play is more important right now.)

Reply:
I think we need to adjust this with the final video. The current video is bad, too long and has a lot of 'dead time' at the beginning.